Post date and time with update.

Dubhghlas opened the pull request:
Display the date and time a post was published just above the article's content. Checks if the article was modified and includes a date and time of the update if found.

Dubhghlas committed:
Post date and time with update.

Display the date and time a post was published just above the article's content. Checks if the article was modified and includes a date and time of the update if found.

joshsmith commented:
Thanks for this PR! I'm a little crammed for time tonight before I get to pull this down and review this, so may not be til mid-week sadly. Will get to it ASAP.

joshsmith commented on web/app/themes/codecorps/templates/content-single.php:
```diff
@@ -13,7 +13,15 @@
       <?php get_template_part('templates/parts/social-share'); ?>
-
+      
+      <p style="font-style: italic;">
```
Can we add a class here instead of the inline styling?

Dubhghlas replied:
That would be best, but I'm not seeing any WordPress CSS files that are accessed on the live site in the repository. I can either remove the styling or make use of the theme's style.css in the theme's root. I only see https://blog.codecorps.org/app/themes/codecorps/dist/styles/main-4aee62f95a.css referenced in the source.

joshsmith commented on web/app/themes/codecorps/templates/content-single.php:
```diff
@@ -13,7 +13,15 @@
+        <strong>Posted</strong>: <?php the_date('F j, Y'); ?> at <?php the_time('g:i a'); ?>
+        <?php if (get_the_time() != get_the_modified_time())
+        { ?>
+          <br><strong>Updated</strong>: <?php the_modified_date('F j, Y'); ?> at <?php the_modified_time('g:i a');
```
Can we also perhaps make use of padding/margin instead of the `br`?

joshsmith replied:
Would also vote on both of these to do date-only.

joshsmith reviewed (commented):
A couple minor comments here!

Dubhghlas committed:
Changes for the night.

Removed some formating. Posted and updated times in their own p-tag.

joshsmith commented:
@Dubhghlas I believe this is what you're looking for https://github.com/code-corps/blog.codecorps.org-site/blob/develop/web/app/themes/codecorps/assets/styles/style.scss

Dubhghlas committed:
Post Date Styles

Added two styles for post dates.

Dubhghlas committed:
Changes for this evening

Encapsulated the two date displays within a div to allow for beautification. Not yet tested.
